### ISSUE TITLE
What is the difference between WeightNormSparsifier and torch.nn.utils.prune.l1_unstructured ?

### ISSUE THREAD

**Issue by mayank64ce** [pytorch/ao#724]

**Comment by supriyar** [comment]
cc @jcaip

**Comment by jcaip** [comment]
Hey @mayank64ce `torch.nn.tils.prun.l1_unstructured` is no longer maintained, so I would recommend using the `WeightNormSparsifier`. The sparsifier also allows for more configs, like block_size, or intra block sparsity. 

Functionally however, they do the same thing - mask dense models to put zeros in the "right" place.

**Comment by mayank64ce** [comment]
I see, in the WeightNormSparsifier, there seem to be some additional parameters other different from l1_unstructured. How can I simulate the same behavior for a give `prune_ratio` .
